### ISSUE TITLE
	NMS complex and  Ndc80 complex 

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#23605]
NMS complex (GO:0031617)

Definition | Parents
-- | --
A supercomplex formed by the association of two subcomplexes (known as MIND and Ndc80 in Schizosaccharomyces) with additional proteins at the kinetochores of condensed nuclear chromosomes. [PMID:16079914]

I don't think we need this complex, it's a bit abstract, and complicates the ontology because it is a connection between inner and outer kinetochore components.

Also, it isn't a "DNA packaging complex" as suggested by the parents.

I suggest to obsolete.

The child term 
Ndc80 complex (GO:0031262)

Definition | Parents
-- | --
An outer kinetochore protein complex that is part of the KMN kinetochore network (also known as the NMS complex) providing the platform with which the plus ends of spindle microtubules directly interact to form stable kinetochore-microtubule attachments.

should be rehoused under "outer kinetochore" [GO:0000940](https://www.pombase.org/term/GO:0000940)

I think you can also edit the definition  of Ndc80 complex (GO:0031262) to
An essential outer kinetochore complex involved in the attachment of  microtubule ends to the chromosomes during mitosis.
adapted from PMID: 28502666 and PMID:34810257

**Comment by raymond91125** [comment]
GO:0031617 NMS complex 10 experimental annotations from PomBase, all of the gene names contain "NMS complex".
There are other NMS complex terms
GO:0044768 NMS complex assembly 0 annotations
GO:1901719 regulation of NMS complex assembly
GO:1901721 positive regulation of NMS complex assembly
GO:1901720 negative regulation of NMS complex assembly (1 IMP annotation from SGD)

**Comment by ValWood** [comment]
OK, don't worry. Its a bit odd because it is really a "network" and consists of components of the outer  and inner kinetochore:
<img width="414" alt="KMN network" src="https://user-images.githubusercontent.com/7359272/179036962-3551884a-5cd1-43d3-a6cd-c7146959dbd1.png">

Some members are not Ndc80 or MIND so we could keep the complex term.

I'm not so sure about the complex assembly terms though (I would  such processes to annotate to inner or outer kinetochore assembly if warranted , and this would be orthogonal). The single annotation is part of the inner kinetochore which forms a platform for the assembly of the kinetchore. I would not annotate to NMS complex assembly though...

**Comment by pgaudet** [comment]
what's the action then?

**Comment by ValWood** [comment]
My preference would be
1. Don't represent the "KML network " aka NMS complex as a single complex.
2. Merge the NMS complex assembly/regulation  terms  into kinetochore assembly  to simplify (I have moved the pombe annotation up, this term was used recently in community curation, but only because they were there)

**Comment by ValWood** [comment]
Actually maybe keep the complex term becasue some of the NMS subunits do not map to either MIND or Ndc80.
We can merge NMS complex assembly though. I don't think it is sensible to separate this from kinetochore assembly because this is basically **the outer kinetochore.**...

**Comment by ValWood** [comment]
Here is another scematic
https://www.nature.com/scitable/content/the-centromere-kinetochore-region-43815/
<img width="422" alt="Screenshot 2022-07-21 at 08 37 22" src="https://user-images.githubusercontent.com/7359272/180157233-0fa659f9-e750-4ce8-bf35-bb232bada2a0.png">

Although Ndc and MIND do interact physically and genetically,  the complex grouping is a bit odd. I think on balance we don't need NMS, and it could merge into outer kinetochore (which is really what it represented historically).

and
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5371998/
The core of the outer kinetochore is a 10-subunit protein assembly known as KMN (for Knl1 complex, Mis12 complex, Ndc80 complex, described in [Figure 2](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5371998/figure/biology-06-00005-f002/))

I don't think we need this extra level if we have outer kinetochore and the sub-complexes. It adds an extra complication.

**Comment by ValWood** [comment]
The NMS "complex" is the outer kinetochore minus the " Mcm21 complex" Called "Mis6-Sim4 complex" in GO, 
- add Mcm21 complex as a synonym of "Mis6-Sim4 complex".

**Comment by raymond91125** [comment]
@ValWood Could you advise what needs to be done still? Thanks.

**Comment by ValWood** [comment]
I still think we should remove NMS complex. It isn't a complex as such and was  defined based on other. characteristics.
PMID: 17035632 
" We found that the 22 mitotic proteins can be classified into three groups: the Mis6-like group, the NMS (Ndc80-Mis12-Spc7) group, and the DASH group, based on their meiotic behavior." I
t is sometimes also called "NMS network".

The Ndc80 and MIND complexes (and others) are sufficient and enable us to organize the terms more cleanly.

1. Merge NMS complex  into "kinetochore" (unnecessary grouping term)
2. Ensure that the existing parent "DNA packaging complex is not transferred) (this is incorrect)
3. Make Ndc80 complex (GO:0031262) part of
GO:0000940 | outer kinetochore (to match existing  definition)
4. Edit the definition of Ndc80 complex (GO:0031262) to
An essential outer kinetochore complex involved in the attachment of microtubule ends to the chromosomes during mitosis.
adapted from PMID: 28502666 and PMID:34810257
5. Make GO:0000444 | MIS12/MIND type complex part of inner kinetochore complex (to match existing  definition)
6. Merge
GO:0044768 NMS complex assembly 0 annotations
GO:1901719 regulation of NMS complex assembly
GO:1901721 positive regulation of NMS complex assembly
GO:1901720 negative regulation of NMS complex assembly (1 IMP annotation from SGD)
into the equivalent kinetochore assembly terms
7. add Mcm21 complex as a synonym of "Mis6-Sim4 complex".

**Comment by raymond91125** [comment]
Thanks.

**Comment by ValWood** [comment]
Thanks @raymond91125  This will be cleaner and reflects the biology better.

**Comment by Antonialock** [comment]
> 5\. Make GO:0000444 | MIS12/MIND type complex part of inner kinetochore complex (to match existing  definition)

Val, why is MIS12/MIND inner kinetochore? in the figure above it is annotated as an outer component?
typo?

other papers also describe MIS12?MIND as outer 
"The outer kinetochore contains the microtubule-binding activity and consists of the essential subcomplexes [Mtw1](http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000032)/Mis12/MIND, [Spc105](http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003061)/Knl-1/Blinkin, [Ndc80](http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001406), [Dam1](http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003345)/DASH/DDD, as well as nonessential proteins such as motors and checkpoint components ([Table 1](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3730914/table/t1/))"
PMC3730914

eg
![image](https://github.com/geneontology/go-ontology/assets/12727506/723c5f14-a498-46e0-a3e2-4b9a221d7803)

**Comment by ValWood** [comment]
hmm some pubs call it inner
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2063780/
All of the hMis12 complex proteins localized to kinetochores at constant levels throughout mitosis, which is consistent with this complex being a stable component of the mitotic inner kinetochore....

**Comment by ValWood** [comment]
mios12 is at the junction between inner and outer. If the distinction is tht inner is assembled during interphase and outer duing mitosis, then it is probably better classified as "inner"

The schematic is  not to realistic.
Consider

<img width="446" alt="Screenshot 2023-05-17 at 12 04 45" src="https://github.com/geneontology/go-ontology/assets/7359272/f7cba586-219f-4183-a08c-a9e8a2f96a00">
<img width="458" alt="Screenshot 2023-05-17 at 12 03 36" src="https://github.com/geneontology/go-ontology/assets/7359272/d7c973f0-8ca5-4ef7-87a3-7b9c2f5fa245">

**Comment by ValWood** [comment]
I can double check....

**Comment by Antonialock** [comment]
I think the text suggests that MIS12 is the part of the outer kinetochore that contacts the inner kinetochore

it looks like inner kinetochore is synonymous with CCAN / Mis6-Sim4 complex 

outer kinetochore consists of MIS12 subcomplex, NDC80 subcomplex, and some other proteins e.g. KNL1

**Comment by Antonialock** [comment]
NMS complex / KMN network synonymous with outer kinetochore?

**Structure of the MIS12 Complex and Molecular Basis of Its Interaction with CENP-C at Human Kinetochores** 
 pmid 27881301
Kinetochores, multisubunit protein assemblies, connect chromosomes to spindle microtubules to promote chromosome segregation. The 10-subunit KMN assembly (comprising KNL1, MIS12, and NDC80 complexes, designated KNL1C, MIS12C, and NDC80C) binds microtubules and regulates mitotic checkpoint function through NDC80C and KNL1C, respectively. MIS12C, on the other hand, connects the KMN to the chromosome-proximal domain of the kinetochore through a direct interaction with CENP-C. The structural basis for this crucial bridging function of MIS12C is unknown. Here, we report crystal structures of human MIS12C associated with a fragment of CENP-C and unveil the role of Aurora B kinase in the regulation of this interaction. The structure of MIS12:CENP-C complements previously determined high-resolution structures of functional regions of NDC80C and KNL1C and allows us to build a near-complete structural model of the KMN assembly. Our work illuminates the structural organization of essential chromosome segregation machinery that is conserved in most eukaryotes

**Comment by ValWood** [comment]
> it looks like inner kinetochore is synonymous with CCAN / Mis6-Sim4 complex

- Mis6-sim4 is narrower than inner kinetochore. Other proteins like Aurora, shugoshin, bub1  alsolocalize to the inner kinetochore, but are not  Mis6-Sim4 complex

- [ ] CCAN = constitutive centromere associated network.  This is CENP-A containing chromatin, plus inner kinetochore. I think it is useful to keep these separate since they have different assembly pathways. I can add as a broad synonym of both.

Since 
https://www.cell.com/molecular-cell/fulltext/S1097-2765(22)00390-2?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1097276522003902%3Fshowall%3Dtrue
is a structure, and recent  and excludes mis12 it seems that this is not a constitutive component after all. 

- [ ] I will switch from inner to outer!

**Comment by ValWood** [comment]
A different opinion:
(Alison Pidoux)

Mis12 is generally considered part of the outer kinetochore. eg see this budding yeast diagram from Sue Biggins   DOI: [10.7554/eLife.37819](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdoi.org%2F10.7554%2Felife.37819&data=05%7C01%7Cvw253%40universityofcambridgecloud.onmicrosoft.com%7C973796aa7f4e45e24abf08db56d8fd90%7C49a50445bdfa4b79ade3547b4f3986e9%7C1%7C0%7C638199262193293152%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=zVDhdn7iGCpkk1ZTOXrvrJYZ%2Fp3LecFfAkTtsaHntjU%3D&reserved=0), and this in fission yeast 

    DOI:[10.1101/cshperspect.a018770](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdoi.org%2F10.1101%2Fcshperspect.a018770&data=05%7C01%7Cvw253%40universityofcambridgecloud.onmicrosoft.com%7C973796aa7f4e45e24abf08db56d8fd90%7C49a50445bdfa4b79ade3547b4f3986e9%7C1%7C0%7C638199262193293152%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=PQcwXm4%2Bg%2FRGtpnwFd9io77C4fDujiqjHc7sbjSowT0%3D&reserved=0)

(Diagrams below)

In mammalian cells etc, some proteins are present at the centromere all through the cell cycle (CENP-A, CENP-C etc), and other (outer kinetochore) proteins join in Mitosis (such at Ndc80).  

But in pombe, almost all proteins stay at the centromere all through the cell cycle, including Mis12, so this isn’t a good way to determine whether they are inner or outer kinetochore. 

(Some pombe proteins, including HJURP/Scm3 and Mis18 actually leave the centromere during part of mitosis.) 

Hope that helps.

**Comment by ValWood** [comment]
The papers Alison refers to 
budding yeast
<img width="939" alt="Screenshot 2023-05-17 at 14 27 17" src="https://github.com/geneontology/go-ontology/assets/7359272/13437c73-6e3e-4370-8bfb-4d1661756695">

fission yeast
<img width="939" alt="Screenshot 2023-05-17 at 14 27 17" src="https://github.com/geneontology/go-ontology/assets/7359272/4a1f4d47-edd1-4dda-a7c9-c457b03bff6d">


but look at the fission yeast subunits  of  Mis12 complex, they are split between inner and outer ?

**Comment by Antonialock** [comment]
> <img alt="Screenshot 2023-05-17 at 14 24 58" width="345" src="https://user-images.githubusercontent.com/7359272/238965487-c71ea160-af63-4d01-9905-9342e31ebce3.png">
> 
> but look at the fission yeast subunits of Mis12 complex, they are split between inner and outer ?

I don't understand what you mean - in this image the fission yeast MIS12 complex are all annotated as outer components (mis12, nsl1, dsn1, nnf1? - they just use higher eukaryote gene names?) 

_"Outside  of CCAN, we find the “outer kinetochore” KMN complex, which is made up of another conserved network of proteins including the Ndc80 and Mis12 protein subcomplexes and Spc7"_

**Comment by ValWood** [comment]
so they are

**Comment by ValWood** [comment]
KMN/NMS nomenclature has been around for a long time.

The network originally referred to a genetic network  the organization was  fully known. That's why I obsoleted NMS complex, it was difficult to accomodate all of the different nomenclatures. 

https://pubmed.ncbi.nlm.nih.gov/17035632/

But I can add "NMS complex"  back as related to "outer kinetochore"

**Comment by Antonialock** [comment]
Ok, I don't know much about this field! But I started looking at it since I was annotating the hsp90-sugt1 complex which helps assemble the MIS12 complex (which I am only doing because of Alaynes request of annotating the plant sugt1 protein....1 paper projects have a tendency to grow!!).....these kinetochore components were annotated a bit inconcistently e.g. mis12 complex in human, but uniprot had "Component of the NMS super complex...." for pombe mis12

**Comment by ValWood** [comment]
So far:
- [ ]  It makes sense the Mis12 complex moves to outer kinetochore.
- [ ] I don't think we need a specific NMS term (it would be an unnecessary intermediate, and didn't originally represent a specific component) but  I will move the synonym from "kinetochore" to be a "narrow synonym" of "outer kinetochore (which will be OK if Mis12 is relocated) (this is why I originally  thought we should remove it).
- [ ]  Add CCAN = constitutive centromere associated network. 
This is CENP-A containing chromatin, plus inner kinetochore. It is probably useful to keep these separate since they have different assembly pathways. I can add as a broad synonym of both terms.
- [ ] GO:0061838   CENP-T-W-S-X complex should be part_of "inner kinetochore"  rather than  part_of "GO:0034080   CENP-A containing chromatin assembly"

In summary there are organismal differences between when some components localize (interphase or mitosis), and so this isn't a useful way to classify inner/outer kinetochore.

**Comment by Antonialock** [comment]
thanks, don't worry about broad synonym for CCAN for me - it was weird not seeing it in go when I searched but I'm sure it would have made sense if I had looked in to it in more detail)

**Comment by ValWood** [comment]
hmm, that makes me think that 
[git7](https://www.pombase.org/gene/SPBC36.12c), [hsp90](https://www.pombase.org/gene/SPAC926.04c)
are unlikely to be directly involved in

**Comment by Antonialock** [comment]
I haven't got to that yet. currently looking at how hsp90-sgt1 stabilises mis12 - but there are other functions to look into (the adenylate cyclase things, innate immunity....maybe it has separate functions - i'll let you know if something looks odd when I get to it.

**Comment by ValWood** [comment]
Takashi wisdom:

One review paper (attached) said this complex locates between inner (CCAN) and outer kinetochores (Knl1 and Noc80C) and is assigned as central.  As Mis12C is part of KNM and it dissociates from the kinetochore during meiotic prophase together with Ndc80C (pdf attached also), I think that Mis12C is not inner and would be classified as outer, but you could assign it as central. 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6756984/

<img width="523" alt="Screenshot 2023-05-18 at 09 06 58" src="https://github.com/geneontology/go-ontology/assets/7359272/a2227670-28c4-41c8-8983-c0e0c0f5c114">



The consensus is definitely outer.

**Comment by Antonialock** [comment]
looks like a term for knl1 complex  part of the outer kinetochore is missing

**Comment by ValWood** [comment]
I can add that, it used to be covered by NMS
So this would be Knl-1 and zwint.

KNL-1/MIS-12/NDC-80 (KMN) network (but I will add as synonyms).

Some of these aren't specific complexes

**Comment by ValWood** [comment]
edited previous comment
